libafl/src/executors/forkserver.rs

```diff
@@ -674,6 +675,110 @@ where
     pub fn coverage_map_size(&self) -> Option<usize> {
         self.map_size
     }
+
+    #[inline]
+    fn execute_input_tracked(&mut self, state: &mut S, input: &TC::Input) -> Result<ExitKind, Error>
```

Comment:
why do we need this method?

Reply:
Guess you can side-step the counting that way.

Reply:
Renamed it to `execute_input` and `execute_input_uncounted`